### DISCUSSION TITLE
也许创建一个issue模版可以改善现在的issue状态

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#362] [Ideas]
这个issue模版可以尽可能的提醒用户：

* 确保自己提出的问题不在以前提过的issue里面（例如列出一个经常被问到的问题清单，比如刚刚提到的树脂问题）；
* 提供自己的系统运行环境；
* 提供自己的操作；
* 提供自己的bot运行日志；
* 提供错误截图

**Comment by Arondight** [comment]
嗯，我写个模板文件

**Reply by mark9804** [reply]
居然有issue form这么高级的东西，但我这里好像不能显示，所有内容还是挤在markdown编辑器里面的
![image](https://user-images.githubusercontent.com/9006264/140651989-0c6309a2-64ba-47bd-aabf-2710e1519004.png)

**Reply by Arondight** [reply]
确实是这样的，我也在看哪里有问题

**Reply by Arondight** [reply]
发了个帖子不知道有老大哥回复么 https://github.com/github/feedback/discussions/7427

**Reply by Arondight** [reply]
试了一万次终于搞好了，现在应该工作了

**Reply by mark9804** [reply]
所以最后是因为文件名不能包含中文吗

**Reply by Arondight** [reply]
是的，应该是个 bug ，因为模板预览没有任何问题
